app/services/statistics/base.rb

```diff
@@ -18,16 +20,22 @@ def id
       @id || count_by.to_s
     end
 
-    def url
-      @url || "statistics/#{id}"
-    end
-
     def search
       "/hosts?search=#{@search}"
     end
 
     def metadata
       {:id => id, :title => title, :url => url, :search => search}
     end
+
+    private
+
+    def build_url
+      taxonomies = {}
+      taxonomies[:location_id] = @location_id if @location_id
+      taxonomies[:organization_id] = @organization_id if @organization_id
```

Comment:
you don't actually need to do this, if you pass a `nil` value for a parameter to the `url_for` helper that parameter will be ignored.

Reply:
right, that's no longer required, fixed